### ISSUE TITLE
A Scenario with two consecutive spaces cant be saved

### ISSUE THREAD

**Issue by MarcelSchmidberger** [HPI-SWA-Teaching/SWT16-Project-07#64]
### Expected Behavior
Unneeded spaces should be ignored.

### Actual Behavior
Two consecutive spaces prevent you from saving the scenario.
![image](https://user-images.githubusercontent.com/29160369/41096737-cdff6672-6a55-11e8-9dd3-c179af423765.png)

### Steps to reproduce error

Enter a second space in the readme from the factory

**Comment by MarcelSchmidberger** [comment]
Estimation 5: Need to check parser

**Comment by Askir** [comment]
I don't think this is actually much of a problem, the popup should be a little nicer and at least tell you that your syntax doesn't fit but I don't think 2 spaces should be allowed. Autocorrection would maybe be nice but honestly at that point we might aswell check for typos etc...

**Comment by MarcelSchmidberger** [comment]
Indentation work so it is ok
